packages/devextreme/js/__internal/ui/date_box/m_time_view.ts

```diff
@@ -9,6 +9,13 @@ import SelectBox from '@js/ui/select_box';
 
 import dateUtils from './m_date_utils';
 
+interface Item {
```

Comment:
We should use other way to define type of Item. We can discuss it, but let's revert all changes in PR that doesn't relate with bug fix. It allow us to cherry pick changes without additional difficulties.